app/services/force/listing_service.rb

```diff
@@ -45,18 +44,13 @@ def units(listing_id)
     end
 
     def sale?(listing)
-      listing.Tenure == 'New sale' || listing.Tenure == 'Resale'
+      self.class.sale?(listing)
     end
 
-    private
+    def self.sale?(listing)
```

Comment:
why have the self method and the non-self method?

Reply:
It's for convenience, so we don't need to instantiate service to check if listing is sale or not. We have this one called also from instantiated service, so I figured it would make sense to have both but in DRY attitude I made one call the other.